server/routes/activities/create-an-activity/handlers/checkAnswers.ts

```diff
@@ -85,6 +85,6 @@ export default class CheckAnswersRoutes {
 
     const response = await this.activitiesService.createActivity(activity, user)
 
-    res.redirect(`confirmation/${response.schedules[0].id}`)
+    res.redirect(`confirmation/${response.id}`)
```

Comment:
Use the activity id and not the schedule id which can be different to the activity id (which is the case in DEV at time of fixing).

I have also testing this locally by manually incrementing the activity schedule sequence so it is not aligned with the activity sequence.